frontend/package.json

```diff
@@ -3,7 +3,11 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
+        "@react-pdf/renderer": "^2.0.12",
         "apexcharts": "^3.26.1",
+        "html2canvas": "*",
+        "jspdf": "^2.3.1",
+        "jspdf-autotable": "^3.5.14",
```

Comment:
Should probably be added to the [BoM](https://docs.google.com/spreadsheets/d/1riRj26jKjfG1dEwkKn4bxvcrr5PahKq9wyz4ao3RDgI/edit#gid=927854276)? @ScoutAtlas